src/supermarket/Gemfile

```diff
@@ -8,6 +8,7 @@ gem "omniauth-chef-oauth2"
 gem "omniauth-github"
 gem "omniauth-oauth2", "~> 1.7.1"
 gem "omniauth-rails_csrf_protection"
+gem "coderay"
```

Comment:
Can we add a comment on this same line that just says what we're using it for. There's a lot of deps in this file and it's helpful to have comments saying why each one is there `# markdown rendering of changelogs` or something like that